README.md

```diff
@@ -1,29 +1,29 @@
-TaskCluster CLI Client
-======================
+# TaskCluster CLI Client
 
-The taskcluster CLI client installed as `taskcluster` provides a collection of
-command line utilities ranging from simple utilities exposing all APIs to
-more special case task like signing-in, admin-scripts or creating of loaners.
+[![Build Status](https://travis-ci.org/taskcluster/taskcluster-cli.svg)](https://travis-ci.org/taskcluster/taskcluster-cli)
 
-Installation
-------------
+## Overview
 
-To install, download the `taskcluster` binary for your platform from the latest
-release on [the releases page](https://github.com/taskcluster/taskcluster-cli/releases).
+TaskCluster CLI is a command-line client offering control and access to
+taskcluster from the comfort of your command-line. It provides utilities
+ranging from direct calls to the specific API endpoints to more complex and
+_useful_ tasks like listing cancelling scheduled runs.
```

Comment:
Weird wording indeed. Maybe look into _advanced_, or _time-saving_. Requesting help from native English staff here!

Also, did you forget a word there?

> like listing _**and**_ cancelling scheduled runs